### PR TITLE
Reintroducing Quarkus Micro Tests

### PR DESCRIPTION
**Pull request by franz1981**
Given that Techempower allow to expose different flavours of the same flamework and Quarkus has a known capability of adapting itself, based on the required level of performance (see https://quarkus.io/blog/magic-control/) we're going to introduce (and re-introduce) 2 different types of Quarkus applications:
- a modularized Quarkus barebone one, which is directly using the core Vertx (+ pgclient) parts which internally are powering what Quarkus usually deliver: `micro` classification and `Raw` ORM
- a refreshed Reactive-Routes + pgclient variant similar to https://github.com/TechEmpower/FrameworkBenchmarks/pull/5728: `fullstack` classification, but `Raw` ORM

The plan is to have a nightly run on the Techempower CI and decide if remove/simplify again any of them, maybe renaming the existing one to better drive users into what each version actually do/what is meant for.

### CODE REVIEW

**Comment by franz1981** [comment]
@NateBrady23 I'm waiting @Sanne to take a quick look: I'll change the naming of these new tests later or now depending the review

**Comment by franz1981** [comment]
Moving it in draft so I can change/improve the naming

File: frameworks/Java/quarkus/muggle/src/main/java/io/quarkus/benchmark/repository/FortuneRepository.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+public class FortuneRepository {
+
+    @Inject
+    PgConnectionPool pgConnectionPool;
+
+    public void findAllSortedFortunes(final Handler<AsyncResult<List<Fortune>>> resultHandler) {
+        pgConnectionPool.pgConnection().selectFortuneQuery()
+                .execute(fortuneRows -> {
+                    if (fortuneRows.succeeded()) {
+                        final List<Fortune> fortunes = new ArrayList<>(fortuneRows.result().size());
```
[the remark this answers is not in the document]
Comment (franz1981) [reply frameworks/Java/quarkus/muggle/src/main/java/io/quarkus/benchmark/repository/FortuneRepository.java:26]: Good point, yep!

File: frameworks/Java/quarkus/benchmark_config.json [hunk 2]
```diff
@@ -47,6 +47,52 @@
         "display_name": "Quarkus, Hibernate Reactive",
         "notes": "",
         "versus": "Netty"
+      },
```
[the remark this answers is not in the document]
Comment (franz1981) [reply frameworks/Java/quarkus/benchmark_config.json:50]: Sure it's a good idea indeed! 
Feel free to send the name of tests too and I will normalize the paths all around.

I still have to add this https://github.com/franz1981/FrameworkBenchmarks/commit/d89a245b578e54efbb11ec78e3deb1b1f974d405 as well

**Comment by franz1981** [comment]
@Sanne i could bring in the enhanced update one for both pgclient and muggle

File: frameworks/Java/quarkus/benchmark_config.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (geoand) [reply frameworks/Java/quarkus/benchmark_config.json:50]: Sure yeah, go ahead and add me as well

**Comment by Sanne** [comment]
sure, why not

**Comment by franz1981** [comment]
These are the whole number of test with the names which would appear on Techempower dashboard:
- Quarkus, Vert.x
- Quarkus, JPA
- Quarkus, Hibernate Reactive
- Quarkus, PostgreSQL

**Comment by Sanne** [comment]
why "JPA" ? We're not using the this API but the Hibernate one

**Comment by franz1981** [comment]
@Sanne fair point, just wanted a short name easier for users: for us the only name is the maven module/docker one (which report everything).

if ORM works better is fine for me, is still short

**Comment by franz1981** [comment]
@dstepanov 
Quarkus is flexible enough to have different parts of it enabled/disabled (see https://quarkus.io/blog/magic-control/) and is built on vertx (at its very core): not only,  we take care to classify these reintroduced tests in the fairest way according to the Techempower classifications (eg no ORM etc etc).

**Comment by dstepanov** [comment]
The composite score will show just the raw Vertx performance labeled as Quarkus, not touching anything around Quarkus. What is the point of it?

My personal opinion is that the code should be written as closely as you would recommend it to be in real life.

**Comment by geoand** [comment]
Both scenarios are real world scenarios.
If there were a notion of weights, then for sure the full blown scenario would be assigned a higher one.

**Comment by franz1981** [comment]
The fact that everything goes under composite score regardless the filter(s) is a bug of techempower's UI, opened at  https://github.com/TechEmpower/FrameworkBenchmarks/issues/8621.
If the filter has worked as expected, depending on the classification and/or presence of ORM, different frameworks should be reported: currently the composite score seems bugged, as reported, while the individual tests ones, nope (but need an additional click on chrome, after applying the filters).

As @geoand said all the cases are real worlds ones, in term of component choice, and given that we had no ORM reactive version in the past, it seems natural to me to have them back now.

**Pull request merged**